snap/hooks/install

```diff
@@ -11,8 +16,10 @@ tar -xJf ${SNAP}/data/mysql.tar.xz -C ${SNAP_COMMON}/lib
 echo "Installing configuration for OpenStack Services"
 for project in neutron nova keystone glance; do
     mkdir -p ${SNAP_COMMON}/etc/${project}/${project}.conf.d
-    cp -r ${SNAP}/etc/${project}/${project}.conf.d/* ${SNAP_COMMON}/etc/${project}/${project}.conf.d
+    cp -r ${SNAP}/etc/${project}/${project}.conf.d/* ${SNAP_COMMON}/etc/${project}/${project}.conf.d || true # Skip conf files that have been moved into templates
```

Comment:
This is kind of an ugly hack. Before we merge, we should probably just move everything in snap-overlay into the templates, and get rid of this line.